src/runtime/ndarray.cc

```diff
@@ -159,7 +167,7 @@ NDArray NDArray::EmptyShared(
     const std::string& name, std::vector<int64_t> shape, DGLDataType dtype,
     DGLContext ctx, bool is_create) {
   NDArray ret = Internal::Create(shape, dtype, ctx);
-  // setup memory content
+  // setup memory content (i.e., allocation)
```

Comment:
Why not just say "allocate memory"?

Reply:
lol... I guess I was trying to be consistent with other 'setup's in `Internal::Create` function. Updated.